Gemfile

```diff
@@ -0,0 +1,5 @@
+ruby '3.2.2'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
+source 'https://rubygems.org'
+gem 'rspec', '~> 3.12'
```

Comment:
- [ ] [REQUIRED] Good job so far! As there are `4 failures` in your spec tests, please fix them 🙏🏼 Your tests should `not` have any failures 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/234972916-341a2f8a-c839-495d-8170-027a280904b1.png)